python/lsst/pipe/drivers/singleFrameDriver.py

```diff
@@ -14,11 +14,13 @@ class SingleFrameDriverTask(BatchParallelTask):
     """
     ConfigClass = SingleFrameDriverConfig
     _DefaultName = "singleFrameDriver"
+    RunnerClass = ButlerInitializedTaskRunner
 
     def __init__(self, *args, **kwargs):
+        butler = kwargs.pop("butler")
```

Comment:
Given that my last proposal has quite a bit more scope than the current issue, I've spun it off into DM-6631.  How about I use `(self, butler=None, *args, **kwds)` for now?  I'll probably tweak that to be consistent with what I'm planning for the others on DM-6631.